examples/UsbMidiConverter/UsbMidiConverter.ino

```diff
@@ -0,0 +1,22 @@
+/*
+ * Simple USBMIDI converter v0.1
+ *
+ * Forwards MIDI messages from USB to MIDI and back
+ * If you short TX and RX pins of arduino, it should act as MIDI loopback
+ *
+ * (c) Tomas 'harvie' Mudrunka 2019
+ */
```

Comment:
Please add a comment with the Arduino board you have tested this code with, also a comment regarding `Serial1` - on some other boards one may have to use `Serial` or `Serial2` instead.

Reply:
I was thinking about this before i commited, but i guess that any arduino with native USB will have to use `Serial1`, because obviously `Serial` is not available on hardware pins. And you can't run this on non-USB arduino, because obviously it's using USBMIDI. On the other hand if your board has Serial2, it's unlikely that it will not have Serial1.

Reply:
Actually, thanks to the underlying v-usb library, you can do USBMIDI on microcontrollers without native USB support, the Digispark board does that too. So on an Arduino with say atmega328, you could have USB MIDI and still have `Serial` as the regular UART port.

So I think noting the kind of board you've used for making the example and a short comment on Serial, as it may be necessary to change it when porting the example to another board, would be helpful :)

Reply:
> Actually, thanks to the underlying v-usb library, you can do USBMIDI on microcontrollers without native USB support

True. I didn't realized, because i haven't tried that yet because of #6 issue :-D

Ok, i've included your suggestions in PR.